charts/node-analyzer/values.yaml

```diff
@@ -23,7 +23,7 @@ image:
   # Secrets must be manually created in the namespace.
   # ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
   #
-  # pullSecrets:
+  pullSecrets: {}
```

Comment:
changes on the pullSecrets here seems out of scope from this PR. May I ask why do we want to change these?

Reply:
@guidobonomi when testing locally the system complained about these not being included in the values.yaml. Generally if you have a field you don't want to have populated in a values.yaml best practice is to leave the field with a blank set rather than uncomment it as it will be potentially excluded from auto-documentation methods.